packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -109,7 +109,7 @@ const options = {
             queries: {
                 retry: false,
                 refetchOnWindowFocus: false,
-                staleTime: 2 * 1000,
+                staleTime: 10 * 1000,
```

Comment:
I realize the staleTime is too short for SSR, when the page loads on client side it is highly likely the data is stale already. Bumping up to 10s.